web/src/OverviewItemView.tsx

```diff
@@ -234,7 +244,7 @@ export function triggerUpdate(name: string, action: string) {
   fetch(url, {
     method: "post",
     body: JSON.stringify({
-      manifest_names: [name],
+      manifestNames: [name],
```

Comment:
is there a server side you have to update?

Reply:
So, I started making these case style changes, but since I'm primarily focused on the front-end of this PR, I was a bit careless with making the changes consistently to the server. I also noticed that there are other places we should use camelCase, which are outside the scope of this PR (e.g., `actionPayload` struct).

I decided to revert the changes and leave it to Maia to review in a separate PR when she comes back.

Reply:
sgtm